### DISCUSSION TITLE
Request for complete chapters

### DISCUSSION THREAD

**Discussion by Omakor1** [DLR-VSDC/IEEE-MVC-2023#13] [General]
Hello,
For proper reference, could you provide all a complete material used for the project (all the chapters used). I only see chapter 3 and 4. That would help my understanding

**Comment by JBRDLR** [comment]
Hi, I hope this helps (as stated in the reader.md):
…
New: the [submitted version of the publication](https://elib.dlr.de/191407/) can be found in our elib repository.
…

Best,

Jonathan

**Reply by Omakor1** [reply]
Thank you @JBRDLR, I have read it and started working on the EMA. I wanted to do a comparison of the EMA for the hybrid battery-fuel and hybrid Battery-Ultracapacitor to see how the ultracapacitor and fuel cell will behave differently. From the references in the pdf, previous edition of this VTS  motor challenge vehicle used battery-ultracapacitor. could you please provide my a FMU model that consist of battery-ultracapacitor or give me an insight? i have tried to figure it out but no headway. this is just for private consumption as i am working the 2023 VTS challenge.  
This will be much appreciated.

**Reply by JBRDLR** [reply]
Dear @Omakor1 at the moment I’m quiet busy, but probably you might be interested in our publication on this topic: 
https://elib.dlr.de/138938/
Probably we consider this in a future project. 
Best, Jonathan
